tailwind.config.js

```diff
@@ -11,6 +13,7 @@ module.exports = {
       },
       fontFamily: {
         sans: ["Inter var", ...defaultTheme.fontFamily.sans],
+        serif: ["Catamaran", ...defaultTheme.fontFamily.sans],
```

Comment:
is this changing the font? I would be hesitant to do that right now in an iterative improvement.

Reply:
Catamaran was the font used in the Figma document.